build/CoreText/dll/CoreText.def

```diff
@@ -33,6 +33,17 @@ LIBRARY CoreText
         ; Core Text Utilities Reference.mm
         CTGetCoreTextVersion
 
+        ; DWriteWrapper Additions
+        _DWriteGetFontFamilyNames
+        _DWriteGetFontNamesForFamilyName
+        _DWriteGetFamilyNameForFontName
+
+        ; Core Text Constant additions
+        kCTFontLabelFontSize
+        kCTFontButtonFontSize
```

Comment:
oops yes

---
In reply to: [78118009](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78118009) [](ancestors = 78118009)